### ISSUE TITLE
AWX Installation on Minishift Unclear

### ISSUE THREAD

**Issue by Sergong** [ansible/awx#5632]
##### ISSUE TYPE

 - Documentation

##### COMPONENT NAME

 - Installer

##### SUMMARY

Installation on Minishift fails with cannot get securitycontextconstraints.security.openshift.io

The AWX installation documentation for minishift mentions the following:

> When using OpenShift for deploying AWX make sure you have correct privileges to add the security context 'privileged', otherwise the installation will fail. The privileged context is needed because of the use of the bubblewrap tool to add an additional layer of security when using containers.

However it is not clear to me how I can ensure I have the correct privileges? It appears that the default system user does not have cluster-admin rights and there seems to be no way that I could find to give it that. So more info on the security pre-requisites would be helpful for a successful installation.

##### ENVIRONMENT
* AWX version: latest
* AWX install method: minishift
* Ansible version:  2.8.5
* Operating System: CentOS 8
* Web Browser: FireFox
* Minishift: 1.34.1

##### STEPS TO REPRODUCE

Install Minishift
Follow the instructions to install AWX

##### EXPECTED RESULTS

AWX install succeeds

##### ACTUAL RESULTS

Installation on Minishift fails with cannot get securitycontextconstraints.security.openshift.io at the cluster scope: no RBAC policy matched

**Comment by shanemcd** [comment]
Hi @Sergong. Does this solve your problem? https://docs.okd.io/latest/minishift/using/addons.html#apply-addons

```
$ minishift addons apply admin-user
$ oc login -u system:admin
```

**Comment by Sergong** [comment]
Hi @shanemcd thanks for the suggestion. 
I wasn't quite sure where in the workflow to apply that, also, I've started using the latest minishift (1.34.2). I've done the following sequence of steps:

1. I've recreated the minishift VM with a `minishift start --cpus=4 --memory=8GB`
2. applied the above suggested addon and enabled it (just applying it leaves it in a disabled state)
3. stopped the minishift vm (as the referenced doc link suggests addons get applied at the start of minishift)
4. it refused to shut down so I simply deleted it and did a `minishift start --cpus=4 --memory=8GB`
5. Then modified the inventory to use the system user (rather than the default of developer)
6. Ran the install.yml playbook (as per the documentation). 

I'm not sure if step 2 enables the addons for all users but the subsequent use of an oc login with the system user suggested to me that its only applied to that user?

After all that, I'm still getting the same error. Which makes me wonder if this combination of versions I'm now using (minishift 1.34.2 + latest AWX running on CentOS 8 with latest updates and KVM) has ever worked for anyone?
